src/incubator/hooks/useHiddenLocation.ts

```diff
@@ -36,13 +36,14 @@ export default function useHiddenLocation<T extends View>(props: HiddenLocationP
 
   const onLayout = useCallback((event: LayoutChangeEvent) => {
     const {width, height} = event.nativeEvent.layout;
-    if (containerRef.current) {
+    if (containerRef.current && hiddenLocation.isDefault) {
```

Comment:
It means that the hidden location is using screen width\height and not the location of the component (that comes from `onLayout`).

Reply:
hmm ok, it wasn't clear. 
What about changing the name to what it actually checks, so the condition is more clear. 
instead of `isDefault`, you do check 
```
if (containerRef.current && !hiddenLocation.wasMeasured) {
```

WDYT?

Reply:
Sounds good, I'll create a new PR with this included (and the files in `incubator/Dialog` and not `incubator/Dialog2)`